caller/train_crf_rnn_loop.sh.dist

```diff
@@ -0,0 +1,18 @@
+export THEANO_FLAGS=mode=FAST_RUN,floatX=float32,device=cuda,force_device=True,exception_verbosity=high
+
+if [ "$#" -lt 1 ]
```

Comment:
Nothing wrong with it, but I thought it might be nice to be able to set it. But since everyone's creating their own version of the script, I guess it's ok to hardcode it